src/Umbraco.Web/Models/Mapping/MemberModelMapper.cs

```diff
@@ -96,9 +98,10 @@ public override void ConfigureMappings(IConfiguration config, ApplicationContext
                 .ForMember(dto => dto.HasPublishedVersion, expression => expression.Ignore());
 
             //FROM MembershipUser TO MemberBasic
-            config.CreateMap<MembershipUser, MemberBasic>()
+            config.CreateMap<MembershipUser, MemberBasic>()                
                 //we're giving this entity an ID of 0 - we cannot really map it but it needs an id so the system knows it's not a new entity
                 .ForMember(member => member.Id, expression => expression.MapFrom(user => int.MaxValue))
+                .ForMember(display => display.Udi, expression => expression.Ignore())

```

Comment:
Ah OK thanks for the difference. IMember our own Members in Umbraco & MembershipUser could use any key (int, guid, something else)for their identifier from a 3rd party Membership Provider like ActiveDirectory or some other weird thing...